README.md

```diff
@@ -32,6 +32,13 @@ version can be found on in the maven repository for
   <artifactId>simpleclient_pushgateway</artifactId>
   <version>0.0.12</version>
 </dependency>
+<!-- Dropwizard exposition-->
```

Comment:
If you're going to add this, add in the rest too. It's currently only the core modules.